README.md

```diff
@@ -1,4 +1,144 @@
+
+Additionally Drafter provide set of Wrappers for serialization, of parsing result, via  [SOS library](https://github.com/apiaryio/sos) into JSON and YAML format.
+
+## Status
+- [Format 1A7](https://github.com/apiaryio/api-blueprint/releases/tag/format-1A7) fully implemented 
+
+## Install
+OS X using Homebrew:
+
+```sh
+$ brew install --HEAD \
+  https://raw.github.com/apiaryio/drafter/master/tools/homebrew/drafter.rb
+```
+
+Other systems refer to [build notes](#build).
+
+## Use
+
+### C++ library
+```c++
+#include "drafter.h"      // Blueprint Parser
+#include "SerializeAST.h" // Blueprint Result Wrapper for serialization
+#include "sosJSON.h"      // Serializer
+
+mdp::ByteBuffer blueprint = R"(
+# My API
+## GET /message
++ Response 200 (text/plain)
+
+        Hello World!
+)";
+
+// Blueprint parsing
+snowcrash::ParseResult<snowcrash::Blueprint> ast;
+drafter::ParseBlueprint(blueprint, 0, ast);
+
+std::cout << "API Name: " << ast.node.name << std::endl;
+
+// Serialization to JSON format
+sos::SerializeJSON serializer;
+serializer.process(drafter::WrapBlueprint(ast.node), std::cout);
+
+```
+
+### C-interface
+
+For purpose of [bindings](#bindings) to other languages Drafter provides very simple C-interface.
+```c
+#include "cdrafter.h"
+
+const char* source = "# My API\n## GET /message\n + Response 200 (text/plain)\n\n        Hello World\n";
+char *result = NULL;
+int ret = drafter_c_parse(source, 0, &result);
+
+printf("Result: %s\n", ret == 0 ? "OK" : "ERROR");
+printf("Serialized JSON result:\n%s\n", result);
+
+free(result); /* we MUST release allocted memory for result */
+```
+
+Refer to [`Blueprint.h`](https://github.com/apiaryio/snowcrash/blob/master/src/Blueprint.h) for the details about the Snow Crash AST and [`BlueprintSourcemap.h`](https://github.com/apiaryio/snowcrash/blob/master/src/BlueprintSourcemap.h) for details about Source Maps tree. See [Drafter bindings](#bindings) for using the library in **other languages**.
+
+
+### Command line tool
+```bash
+$ cat << 'EOF' > blueprint.apib
+# My API
+## GET /message
++ Response 200 (text/plain)
+
+        Hello World!
+EOF
+
+$ drafter blueprint.apib 
+_version: 2.1
+metadata:
+name: "My API"
+ ...
+```
+
+Refer to [AST Serialization Media Types](https://github.com/apiaryio/api-blueprint-ast) for the details on serialized media types. See [parse feature](features/parse.feature) for the details on using the `drafter` command line tool.
+
+## Build
+1. Clone the repo + fetch the submodules:
+
+	```sh
+	$ git clone --recursive git://github.com/apiaryio/drafter.git
+	$ cd drafter
+	```
+
+2. Build & test Drafter:
+
+	```sh
+	$ ./configure
+	$ make test
+	```
+
+	To include integration tests (using Cucumber) use the `--include-integration-tests` flag: 
+
+	```sh
+	$ ./configure --include-integration-tests
+	$ make test
+	```
+	
+We love **Windows** too! Please refer to [Building on Windows](https://github.com/apiaryio/snowcrash/wiki/Building-on-Windows).
```

Comment:
Can you change the link here to `https://github.com/apiaryio/drafter/wiki/Building-on-Windows`? I copied the wiki page to drafter and edited it.